### ISSUE TITLE
When creating a PR containing only doc changes

### ISSUE THREAD

**Issue by StevenPuttemans** [opencv/opencv#7115]
Wouldn't it make sense to disable the buildbots for the source code if only files from doc folder are edited? This would speed up the build of the docs for verification.

**Comment by sturkmen72** [comment]
is it possible now? see #10646

**Comment by alalek** [comment]
@sturkmen72 Right. Just add these lines into your PR description before pushing changes into source branch:
- `**WIP**` blocks automatic scheduling of builds
- `force_builders=Docs` forces build for `Doc` builder only.

It is fine for intermediate testing, but before merging this may be not enough. In general it is hard to detect "doc changes" only - sometimes parameters are renamed outside of comments. Also buildbot master (controller, which schedules builds) doesn't work with the code at all for now.
